guides/updates/administration/vue3.md

```diff
@@ -0,0 +1,129 @@
+
+- (Vue 3 migration guide)[https://v3-migration.vuejs.org]
+- (Vue 3 breaking changes)[https://v3-migration.vuejs.org/breaking-changes/]
+- (Vue router migration guide)[https://router.vuejs.org/guide/migration/]
+- (Vue test utils migration guide)[https://test-utils.vuejs.org/migration/]
+
+## Step-by-step guide
+::: warning
+Depending on when you read this document, you still need to enable the `VUE3` feature flag.
+To be safe, enable the `VUE3` feature flag and restart your watcher.
+:::
+
+To follow along, you should have the following:
+
+- the latest Shopware `trunk` or an official release candidate
+- installed and activated your plugin
+- a running administration watcher (`composer run watch:admin`)
+
+### Update your plugin npm dependencies
+
+Make sure to align your `package.json` dependencies with the administration.
+
+### Verify the Admin is using Vue 3
+
+If you evaluate the following expression in the browser dev tools, the value `true` must be returned.
+
+```javascript
+Shopware.Application.view.vue3
+```
+
+### Check your templates
+
+For your templates to work correctly, follow the following in no specific order:
+
+- Replace all `sw-field` usages with the corresponding components.
+- (Check all v-models)[https://v3-migration.vuejs.org/breaking-changes/v-model.html]
+- (Check for deprecated slot syntax)[https://eslint.vuejs.org/rules/no-deprecated-slot-attribute.html]
+- (Check router-view transition combinations)[https://router.vuejs.org/guide/migration/#-router-view-keep-alive-and-transition-]
+- (Check your key attributes)[https://v3-migration.vuejs.org/breaking-changes/key-attribute.html]
+
+### Check your code
+
+Most of your code should be unaffected by the update. You can start by searching for `this.$`. The usage of `this.$` is an indicator of Vue's internal API. These calls are very likely to break except for `this.$tc`.
+
+If you have a lot of Vue internal API calls, check out the (Known issues section)[#known-issues].
+The best way to find errors is to test your application thoroughly, either by hand or automated.
+
+## Known issues
+
+### Lifecycle hooks
+Lifecycle hooks such as `@hook:mounted` may be triggered multiple times if the component is loaded asynchronously. Vue 3 will emit the hook for the `AsyncComponentWrapper` and the underlying component. Expect lifecycle hooks to be triggered more than once.
+
+### Using slots programmatically
+It is no longer sufficient to check if `this.$slots` has a property with the slot name to see if that slot exists. Instead, you must verify if your `slotName` contains an actual `v-node`.
+
+### this.$parent
+`this.$parent` is prone to errors because Vue 3 wraps the `AsyncWrapperComponent` around asynchronous components. Leading to the virtual dom tree to differ from Vue 2 to Vue 3. Where in Vue 2, a `this.$parent` call was successful, in Vue 3, a `this.$parent.$parent` may be necessary.
```

Comment:
Should we make a note to maybe prefer a different method of inter component communication?